src/pseudopeople/interface.py

```diff
@@ -27,6 +24,7 @@
     configure_logging_to_terminal,
     get_engine_from_string,
     get_state_abbreviation,
+    to_string,
     to_string_as_integer,
     to_string_preserve_nans,
```

Comment:
Can these methods be integrated into `to_string`?

Reply:
Yep!